pkg/operator/revisioncontroller/revision_controller.go

```diff
@@ -61,7 +71,7 @@ func NewRevisionController(
 	configMaps []RevisionResource,
 	secrets []RevisionResource,
 	kubeInformersForTargetNamespace informers.SharedInformerFactory,
-	operatorClient v1helpers.StaticPodOperatorClient,
+	operatorClient LatestRevisionClient,
```

Comment:
are we going to add a new impl for `openshift-server`

Reply:
Already done.